CODEOWNERS

```diff
@@ -187,6 +190,7 @@
 /gallery/workbook/microsoft.mobilenetwork-mobilenetwork-sites.json @microsoft/azure-private-mec-workbooks
 /gallery/workbook/microsoft.apimanagement-service.json @microsoft/azure-api-management
 /gallery/workbook/microsoft.experimentation-experimentworkspaces.json @microsoft/exp-devs
+/gallery/workbook/microsoft.machinelearningservices-workspaces @microsoft/gen-ai-insights
```

Comment:
you added 2 gallery files, so you should have 2 gallery entries in CODEOWNERS

Reply:
Done